File: Typescript/Practice/26072022/easy/easy_drinkSorting.ts
#!/usr/bin/env npx ts-node

function sortDrinkByPrice (arr: Array<{name:string, price:number}>)
  : Array<{name:string, price:number}> {
  arr.sort((firstItem, secondItem) => firstItem.price - secondItem.price);
  return arr;
};

const drinks: Array<{name:string, price:number}> = [
  {name: "lemonade", price: 50},
  {name: "lime", price: 10}
];

console.log(sortDrinkByPrice(drinks));

File: Typescript/Practice/26072022/easy/easy_drinkSorting.py
#!/usr/bin/env python

def sort_drinks_by_price(drinks):
  drinks.sort(key = lambda x: x['price'])
  return drinks

drinks = [
  {"name": "lemonade", "price": 50},
  {"name": "lime", "price": 10}
]

print(sort_drinks_by_price(drinks))
